<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsComposerScaleBarWidgetBase</class>
 <widget class="QWidget" name="QgsComposerScaleBarWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>537</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Barscale Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>307</width>
        <height>714</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Scale bar</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="mSegmentSizeLabel">
         <property name="text">
          <string>Segment size (map units)</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>mSegmentSizeSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QDoubleSpinBox" name="mSegmentSizeSpinBox">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="maximum">
          <double>9999999999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="mMapUnitsPerBarUnitLabel">
         <property name="text">
          <string>Map units per bar unit</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>mMapUnitsPerBarUnitSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QDoubleSpinBox" name="mMapUnitsPerBarUnitSpinBox">
         <property name="maximum">
          <double>9999999999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="mNumbereOfSegmentsLabel">
         <property name="text">
          <string>Number of segments</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>mNumberOfSegmentsSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QSpinBox" name="mNumberOfSegmentsSpinBox"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="mSegmentsLeftLabel">
         <property name="text">
          <string>Segments left</string>
         </property>
         <property name="buddy">
          <cstring>mSegmentsLeftSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QSpinBox" name="mSegmentsLeftSpinBox"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="mStyleLabel">
         <property name="text">
          <string>Style</string>
         </property>
         <property name="buddy">
          <cstring>mStyleComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QComboBox" name="mStyleComboBox"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="mMapLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Map</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>mMapComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QComboBox" name="mMapComboBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="mHeightLabel">
         <property name="text">
          <string>Height (mm)</string>
         </property>
         <property name="buddy">
          <cstring>mHeightSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QSpinBox" name="mHeightSpinBox"/>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="mLineWidthLabel">
         <property name="text">
          <string>Line width</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>mLineWidthSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QDoubleSpinBox" name="mLineWidthSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.200000000000000</double>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="mLabelBarSpaceLabel">
         <property name="text">
          <string>Label space</string>
         </property>
         <property name="buddy">
          <cstring>mLabelBarSpaceSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QDoubleSpinBox" name="mLabelBarSpaceSpinBox"/>
       </item>
       <item row="18" column="0">
        <widget class="QLabel" name="mBoxSpaceLabel">
         <property name="text">
          <string>Box space</string>
         </property>
         <property name="buddy">
          <cstring>mBoxSizeSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="19" column="0">
        <widget class="QDoubleSpinBox" name="mBoxSizeSpinBox"/>
       </item>
       <item row="20" column="0">
        <widget class="QLabel" name="mUnitLabelLabel">
         <property name="text">
          <string>Unit label</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>mUnitLabelLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="21" column="0">
        <widget class="QLineEdit" name="mUnitLabelLineEdit"/>
       </item>
       <item row="22" column="0">
        <widget class="QPushButton" name="mFontButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Font...</string>
         </property>
        </widget>
       </item>
       <item row="23" column="0">
        <widget class="QPushButton" name="mColorPushButton">
         <property name="text">
          <string>Color...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
